lib/dmv.rb

```diff
@@ -1,16 +1,40 @@
+<<<<<<< HEAD
```

Comment:
Anytime you see <<<<< and ===== symbols, those indicate the presence of a merge conflict and technically the merge conflict is not resolved until these symbols are cleared. In its current state, this class is not stable/usable until that merge conflict is resolved. Notice that this top section is the initial starter code for this class.